### DISCUSSION TITLE
how navigation only right to left (x position) ?

### DISCUSSION THREAD

**Discussion by Nazeh-Taha** [xyflow/xyflow#1288] [Q&A] [answered]
how navigation only right to left (x position)
![Annotation 2021-06-17 132156](https://user-images.githubusercontent.com/54739625/122391896-0fa07380-cf7c-11eb-9a23-b096451c7935.jpg)

**Comment by moklick** [comment, marked as answer]
You could try `paneMovable={false}` in combination with `panOnScrollMode="horizontal"`. Does it work?

**Reply by Nazeh-Taha** [reply]
@moklick  translateExtent prop works fine, Thanks .
